iosHackthonApp/iosHackthonApp/Resources/Strings/Strings.swift

```diff
@@ -9,31 +9,48 @@
 import Foundation
 
 struct Strings {
-    struct LandingScreen {
-        static let heading = "Welcome!"
-        static let subheading = "List a meal to help out someone in your local community or browse the meals that others have added"
-        static let plusButtonText = "List a Meal"
-        static let findButtonText = "Find a Meal"
-
-        struct Images {
-            static let plus = "plus.app"
-            static let find = "magnifyingglass"
-        }
-    }
-
-    struct AddMealScreen {
-        struct CollectionAlert {
-            static let message = "When someone arrives to collect this meal they will quote the above code"
-        }
-    }
-
-    struct Common {
-        struct ErrorAlert {
-            static let message = "There was an error creating your meal.\n please try again"
-        }
-        static let ok = "Ok"
-        static let sorry = "Sorry!"
-    }
+	struct LandingScreen {
+		static let heading = "Welcome!"
+		static let subheading = "List a meal to help out someone in your local community or browse the meals that others have added"
+		static let plusButtonText = "List a Meal"
+		static let findButtonText = "Find a Meal"
+
+		struct Images {
+			static let plus = "plus.app"
+			static let find = "magnifyingglass"
+		}
+	}
+
+	struct AddMealScreen {
+		static let title = "Title"
+		static let additionalInfo = "Additional Information"
+		static let quantity = "Quantity"
+		static let temperature = "Temperature"
+		static let hot = "Hot"
+		static let cold = "Cold"
+		static let availableFrom = "Available From"
+		static let useBy = "Use By"
+		static let address = "Address"
+		static let addMeal = "Add A Meal"
+
+		struct Images {
+			static let hotFood = "flame"
+			static let coldFood = "snow"
+			static let location = "location"
```

Comment:
Could you move these to a Common > Images, I will reuse them on ListScreen

Reply:
Done